india_banking/overrides/journal_entry.py

```diff
@@ -0,0 +1,147 @@
+import frappe
+from erpnext.accounts.doctype.accounting_dimension.accounting_dimension import (
+	get_accounting_dimensions,
+)
+from frappe.query_builder import DocType
+from frappe.query_builder.functions import Sum
+
+
+@frappe.whitelist()
+def make_payment_order(source_name, target_doc=None, args=None):
+	from frappe.model.mapper import get_mapped_doc
+
+	def update_bank_entry(source, target):
+		JournalEntryAccount = DocType("Journal Entry Account")
+		BankAccount = DocType("Bank Account")
+
+		select_field = [
+			"name",
+			"account",
+			"cost_center",
+			"project",
+			"debit as amount",
+			"party as employee",
+			"party_type",
+			"parent as journal",
+		]
+		select_field.extend(get_accounting_dimensions())
+
+		# Build the query
+		query = (
+			frappe.qb.from_(JournalEntryAccount)
+			.join(BankAccount)
+			.on(JournalEntryAccount.party == BankAccount.party)
+			.select(
+				*[
+					getattr(JournalEntryAccount, field.split(" as ")[0]).as_(
+						field.split(" as ")[1]
+					)
+					if " as " in field
+					else getattr(JournalEntryAccount, field)
+					for field in select_field
+				],
+				BankAccount.name.as_("party_bank_account"),
+			)
+			.where(
+				(JournalEntryAccount.parent == source.name)
+				& (JournalEntryAccount.party_type == "Employee")
```

Comment:
Does it mean, we do not allow any supplier, customer bank entry type journal entries?